lib/animina_web/components/top_navigation_components.ex

```diff
@@ -431,6 +431,21 @@ defmodule AniminaWeb.TopNavigationCompontents do
     """
   end
 
+  def hibernated_user_banner(assigns) do
+    ~H"""
+    <div
+      :if={@current_user && make_sure_user_state_is_atom(@current_user) == :hibernate}
+      class="mx-4 mb-4 p-2 text-center rounded-md  bg-blue-500 text-white"
+    >
+      <%= with_locale(@language, fn -> %>
+        <%= gettext(
+          "Your Account is Hibernated , This means users can't see your profile or send you messages. You can activate your account by ensuring you have a profile picture and an about me story with an image"
```

Comment:
That text assumes that the profile picture is missing (it doesn't but kind of it does) which doesn't make sense. A user might just hibernate his account for what ever reason. The text should be "This account is hibernated. Other users can't see it or send messages to you. Visit the `Change Visibility` page to activate it again." The "Change Visibility" part should be a link to that page.

On THAT page you have to make sure that one can not activate the account if there is not profile picture or about me story.